.env.example

```
@@ -0,0 +1,238 @@
+REDIS_CONNECTION_STRING="localhost:6379"
+
+# =============================================================================
+# AUTHENTICATION & SECURITY
+# =============================================================================
+JWT_SECRET_KEY="your-super-secret-jwt-key-change-this-in-production"
+JWT_ISSUER="https://localhost:5000"
+JWT_AUDIENCE="barq-users"
+JWT_EXPIRY_HOURS="24"
+
+# Cookie Authentication
+COOKIE_AUTH_ENABLED="true"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)